stories/utils/adsense-data-zeroing.js

```diff
@@ -1,3 +1,29 @@
+/**
+ * AdSense Settings components.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Returns an AdSense report with zeroed data.
+ *
+ * @since 1.77.0
```

Comment:
This should be `@since n.e.x.t`, the `n.e.x.t` will be replaced with the release number when the release is created.

```suggestion
 * @since n.e.x.t
```